src/lightning_app/core/api.py

```diff
@@ -69,43 +67,33 @@ class SessionMiddleware:
 # in a serverless architecture
 
 
-class UIRefresher(Thread):
-    def __init__(self, api_publish_state_queue, api_response_queue, refresh_interval: float = 0.1) -> None:
+class APIRefresher(Thread):
+    def __init__(self, api_queue) -> None:
         super().__init__(daemon=True)
-        self.api_publish_state_queue = api_publish_state_queue
-        self.api_response_queue = api_response_queue
+        self.api_queue = api_queue
         self._exit_event = Event()
-        self.refresh_interval = refresh_interval
 
     def run(self):
         # TODO: Create multiple threads to handle the background logic
         # TODO: Investigate the use of `parallel=True`
         try:
             while not self._exit_event.is_set():
                 self.run_once()
-                # Note: Sleep to reduce queue calls.
-                sleep(self.refresh_interval)
```

Comment:
Haven't we decided to add a longer sleep time here given that we are now pulling all the deltas?

Reply:
No, we want this to be as small as possible.